### ISSUE TITLE
[Request] Disable Steam Input

### ISSUE THREAD

**Issue by Immersion95** [doZennn/steam-rom-manager#562]
Thanks for allowing to use Steam template for the controller, would it be possible to have the option to disable Steam Input in the preset  ?

Citra/Yuzu/Ryujinx need direct access to the controller to have SDL working which allows to use HD rumble, Gyro, etc.... I currently have to disable it for each game :p.

**Comment by JupiterJesus** [comment]
I second this feature request, specifically for yuzu

**Comment by sqrlmstr5000** [comment]
Yea, running most emulators through Steam it's suggested to Disable Steam Input and Disable Use Desktop Configuration in Launcher

**Comment by dathanhansell** [comment]
Would love this option, hate manually disabling each time i add a game

**Comment by sqrlmstr5000** [comment]
I've looked into see if this is even possible and I'm not seeing any reference to it. I see no reference to controller configs in anything related to editing shortcuts.vdf. This file is binary so you it's not like you can just open it and browse through it. Steam must store the controller configs somewhere else...

https://developer.valvesoftware.com/wiki/Steam_Library_Shortcuts
https://github.com/tirish/steam-shortcut-editor
https://github.com/Corecii/steam-binary-vdf-ts

**Comment by sqrlmstr5000** [comment]
Looks like it might be in `localconfig.vdf` which is a JSON file

I edited a Non-Steam Game to Disable Steam Input and it added this section to `localconfig.vdf`. 
```
		"-1124853874"
		{
			"UseSteamControllerConfig"		"0"
			"SteamControllerRumble"		"-1"
			"SteamControllerRumbleIntensity"		"320"
		}
```

**Comment by MwenDavo** [comment]
Really hoping this gets implemented, having to manually change it for every games isn't the best way to do it.

**Comment by dathanhansell** [comment]
Judging by tsquillario's findings, it seems relatively trivial to implement, so hopefully it does get added as a feature.

**Comment by cbartondock** [comment]
Working on this, it will be in the next release.

**Comment by cbartondock** [comment]
Implemented in 2.4.20
